react/components/SearchHeader.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage, injectIntl } from 'react-intl'
+
+import Dropdown from '@vtex/styleguide/lib/Dropdown'
```

Comment:
`import { Dropdown } from 'vtex.styleguide'`

In manifest.json `vtex.styleguide: 4.x`